DifferentiationInterface/ext/DifferentiationInterfaceGTPSAExt/twoarg.jl

```diff
@@ -0,0 +1,195 @@
+## Pushforward
+
+# Input: Contains either a single pre-allocated initial TPS
+# or a vector of pre-allocated TPSs.
+#
+# Output: Contains a vector of pre-allocated TPSs
+struct GTPSATwoArgPushforwardPrep{X,Y} <: DI.PushforwardPrep
+    xt::X
+    yt::Y
+end
+
+function DI.prepare_pushforward(
+    ::F, y, backend::AutoGTPSA{D}, x, tx::NTuple, ::Vararg{DI.Context,C}
```

Comment:
If I'm not mistaken, `DI.Cache` is not supported as a `Context` subtype, so I would replace all annotations `DI.Context` with `DI.Constant`?